libs/pcapplusplus/Makefile

```
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2021 Michal Hrusecky <michal.hrusecky@turris.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pcapplusplus
+PKG_VERSION:=21.05
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=f7bc2caea72544f42e3547c8acf65fca07ddd4cd45f7be2f5132dd1826ea27bb
+PKG_SOURCE_URL=https://github.com/seladb/PcapPlusPlus/archive/v21.05.tar.gz?
```

https://github.com/seladb/PcapPlusPlus/releases/download/v21.11/pcapplusplus-21.11-source-linux.tar.gz is not usable?

---

```suggestion
PKG_SOURCE_URL:=https://github.com/seladb/PcapPlusPlus/archive/v21.05.tar.gz?
```

---

How about using `$(PKG_VERSION)` instread of hard-coding the version  here?
```suggestion
PKG_SOURCE_URL:=https://github.com/seladb/PcapPlusPlus/archive/v$(PKG_VERSION).tar.gz?
```